src/timeline/components/metrics-components/FullSessionMetrics.jsx

```diff
@@ -25,22 +29,22 @@ const FullSessionMetrics = () => {
             <p># Tasks Completed: {sessionMetrics.numTasks}</p>
           </div>
           <div>
-            <p>Tab Switches per min: {sessionMetrics.tabSwitchRate}</p>
+            <p>Tab Switches per min: {sessionMetrics.tabSwitchRate.toFixed(2)}</p>
```

Comment:
what's toFixed for?

Reply:
Basically just truncates the decimal (so 2.333333 -> 2.33)